ocaml/testsuite/tests/backtrace/backtrace_c_exn.ml

```diff
@@ -45,7 +45,13 @@ let () =
      native;
    }
  }{
-   reference = "${test_source_directory}/backtrace_c_exn.byte.reference";
    runtime5;
+   {
+     reference = "${test_source_directory}/backtrace_c_exn.byte.reference";
+     bytecode;
+   }{
+     reference = "${test_source_directory}/backtrace_c_exn.byte.reference";
```

Comment:
Is `byte` correct on this line?
(if so, `reference` can possibly
be factored out)

Reply:
I think this is right since the CI passes, but I've renamed this file.